e2e/pages/app/dashboard-widgets/dashboard-widgets.testpage.component.ts

```diff
@@ -0,0 +1,70 @@
+import { AfterViewInit, Component, TemplateRef, ViewChild } from '@angular/core';
+import {
+    ActionConfig,
+    ActionStatus
+} from '../../../../src/components/dashboard-widgets/interfaces/actions-widget.interface';
```

Comment:
If you are still having build issues, you should correct this to import from @ux-aspects/ux-aspects.